tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -389,9 +389,22 @@ def test_parse_bad_trailer_file2(bad_trailer_file_2):
     assert length_error.content_type is None
     assert length_error.object_id is None
 
+    trailer_error_3 = trailer_errors[3]
+    assert trailer_error_3.error_type == ParserErrorCategoryChoices.PRE_CHECK
+    assert trailer_error_3.error_message == ('The value: trash, does not follow the YYYYMM format for '
+                                             'Reporting Year and Month.')
+    assert trailer_error_3.content_type is None
+    assert trailer_error_3.object_id is None
+
+    trailer_error_4 = trailer_errors[4]
+    assert trailer_error_4.error_type == ParserErrorCategoryChoices.PRE_CHECK
+    assert trailer_error_4.error_message == 'T1trash contains blanks between positions 8 and 19.'
```

Comment:
What is trash? typo?

Reply:
i think `T1trash` is the entire/literal header record for one of our test files